src/benchmarks/micro/corefx/System.Collections/Remove/CreateAddAndRemove.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+using System.Collections.Generic;
+
+namespace System.Collections
+{
+    // the concurrent collections are covered with benchmarks in Add_Remove_SteadyState.cs
+    [BenchmarkCategory(Categories.CoreFX, Categories.Collections, Categories.GenericCollections)]
+    [GenericTypeArguments(typeof(int))] // value type
+    [GenericTypeArguments(typeof(string))] // reference type
+    public class CreateAddAndRemove<T>
+    {
+        [Params(Utils.DefaultCollectionSize)]
+        public int Size;
+
+        private T[] _keys;
+
+        [GlobalSetup]
+        public void Setup() => _keys = ValuesGenerator.ArrayOfUniqueValues<T>(Size);
+
+        [Benchmark]
+        public List<T> List()
+        {
+            List<T> list = new List<T>();
+            foreach (T uniqueKey in _keys)
+            {
+                list.Add(uniqueKey);
+            }
+            foreach (T uniqueKey in _keys)
+            {
+                list.Remove(uniqueKey);
+            }
+            return list;
+        }
+
+        [Benchmark]
+        public LinkedList<T> LinkedList()
+        {
+            LinkedList<T> linkedList = new LinkedList<T>(_keys);
```

Comment:
Internally this just calls `AddLast` a bunch. For consistency perhaps make it look like all the other tests? (Saving someone else looking up the source like I just did..)